### DISCUSSION TITLE
problem with installing Subpackages micom.data ...

### DISCUSSION THREAD

**Discussion by AlfredoRA** [micom-dev/micom#97] [Q&A]
Hi! I also have some problems with installing MICOM. In my case:

Subpakages are not installed by default when I write : "pip install micom" ... in my cobra environment, so when I use "from micom.data import test_data" (for example) an error says:
" from: can't read /var/mail/micom.data"

I will appreciate any help!

**Comment by cdiener** [comment]
I don't think you are running this from a Python interpreter. The `from micom.data import test_data` has to be run from within Python an dnot in your bash shell. So you would first type `python`+Enter and then write your code.

**Reply by AlfredoRA** [reply]
Thanks. When I run this lines from python an error appears "" ImportError: cannot import name 'test_data' from 'micom.data' (/home/alfredo/.local/lib/python3.7/site-packages/micom/data/__init__.py) "" It seem like a versions problem? or why is searching test_data from __init__.py? micom.data do not appears anywhere

**Reply by cdiener** [reply]
Because test_data gets defined in the init file. Could be a version info problem or a botched installation. What version of MICOM are you running?

**Reply by AlfredoRA** [reply]
versión 0.30.4 ; I first installed a conda environment for micom and then micom

**Comment by cdiener** [comment]
That is the most recent version. Did you activate the conda environment before you installed micom with pip? Because your import is definitely not importing a package from conda...
